filter_logs_to_metrics: document field_value usage for counters

databus23 opened the pull request:
This documents a feature added with this PR: https://github.com/fluent/fluent-bit/pull/10055

<!-- This is an auto-generated comment: release notes by coderabbit.ai -->
## Summary by CodeRabbit

* **Documentation**
  * Added a new derived counter metric, "total_duration", sourced from the duration field and documented with a description.
  * Updated Prometheus sample output to include the new metric and adjusted sample values for existing counter examples.

<!-- review_stack_entry_start -->

[![Review Change Stack](https://storage.googleapis.com/coderabbit_public_assets/review-stack-in-coderabbit-ui.svg)](https://app.coderabbit.ai/change-stack/fluent/fluent-bit-docs/pull/1584?utm_source=github_walkthrough&utm_medium=github&utm_campaign=change_stack)

<!-- review_stack_entry_end -->
<!-- end of auto-generated comment: release notes by coderabbit.ai -->

databus23 committed:
filter_logs_to_metrics: document field_value usage for counters

This documents a feature added with this PR: https://github.com/fluent/fluent-bit/pull/10055

Signed-off-by: Fabian Ruff <fabian.ruff@sap.com>

alexakreizinger reviewed (commented):
hi @databus23, thanks for submitting a docs PR! I just suggested a few edits for clarity and consistency.

databus23 committed:
Apply suggestions from code review

Co-authored-by: Alexa Kreizinger <alexakreizinger@gmail.com>
Signed-off-by: Fabian Ruff <fabian@progra.de>

alexakreizinger reviewed (approved):
awesome, thank you :) I'm approving this, but I won't merge anything since it sounds like we're still waiting on a different code PR—feel free to merge this one when the time comes.

esmerel committed:
Merge branch 'master' into logs_to_metrics-pr-10055

Signed-off-by: Lynette  Miles <6818907+esmerel@users.noreply.github.com>
Signed-off-by: Eric D. Schabell <eric@schabell.org>

eschabell commented on pipeline/filters/log_to_metrics.md:
```diff
@@ -107,6 +107,15 @@ pipeline:
+    metric_mode        counter
+    metric_name        total_duration 
+    metric_description This metric counts the total time spent across all messages
+    value_field        duration
```
Fixed.

eschabell committed:
docs: pipeline: filters: log_to_metrics: sync YAML and classic conf counter examples

  - Add total_duration counter filter to fluent-bit.yaml example to match
    fluent-bit.conf
  - Fix indentation in second [FILTER] block from 4-space to 2-space
  - Apply Title_Case to all keys in the classic conf counter example

  Applies to #1584

Signed-off-by: Eric D. Schabell <eric@schabell.org>

eschabell committed:
Merge branch 'master' into logs_to_metrics-pr-10055

Signed-off-by: Eric D. Schabell <eric@schabell.org>

eschabell reviewed (approved):
@databus23 waiting now on code PR merging.
